utils/biospecimen.js

```diff
@@ -215,7 +215,8 @@ const biospecimenAPIs = async (req, res) => {
             const masterSpecimenId = req.query.masterSpecimenId;
             if(!masterSpecimenId) return res.status(400).json(getResponseJSON('Bad request!', 400));
             const { searchSpecimen } = require('./firestore');
-            const response = await searchSpecimen(masterSpecimenId, siteCode);
+            const allSitesFlag = (req.query.allSitesFlag) ? true : false
```

Comment:
No changes requested, just a comment:
This shortened version might be worth considering: `const allSitesFlag = !!req.query.allSitesFlag`

Reply:
I have never used !! let me look into it.